src/Microsoft.SqlTools.ServiceLayer/Utility/SqlScriptFormatter.cs

```diff
@@ -260,6 +337,12 @@ private static string FormatBinary(object value)
             return "0x" + BitConverter.ToString(bytes).Replace("-", string.Empty);
         }
 
+        private static bool HasNextCharacter(string haystack, char needle, int position)
```

Comment:
This is actually a very popular naming convention in PHP: http://php.net/manual/en/function.strpos.php